search-app/src/cmr/search/results_handlers/umm_json_results_handler.clj

```diff
@@ -101,5 +103,7 @@
   [context query results]
   (let [granule-counts-map (:granule-counts-map results)
         items (:items results)
-        results (assoc results :items (add-granule-count-to-items items granule-counts-map))]
+        results (assoc results :items (if (contains? (set (:result-features query)) :granule-counts)
+                                        (add-granule-count-to-items items granule-counts-map)
```

Comment:
I assume this will cause the granule-count parameter to drop out entirely? No more zero values when parameter was unrequested and granules do actually exist?

Reply:
that is correct

Reply:
we talked on slack, yes, it looks good.